mock/saves/1.14.4/mock.txt

```diff
@@ -0,0 +1,512 @@
+1433 0 0 0 Block{minecraft:obsidian}
```

Comment:
That's because the Minecraft `Block#toString` has different behavior in 1.14 and 1.12. 

For 1.12, this naming is just a hint to read.
For 1.14, the naming is the data to validate.